backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/invoice/isCompleteInvoice.feature

```diff
@@ -0,0 +1,120 @@
+@from:cucumber
+@dev:runThisOne
+Feature: completeInvoice option when processing invoice candidates
+
+  Background:
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And set sys config boolean value true for sys config SKIP_WP_PROCESSOR_FOR_AUTOMATION
+    And set sys config boolean value true for sys config de.metas.report.jasper.IsMockReportService
+    And metasfresh has date and time 2021-12-21T13:30:13+01:00[Europe/Berlin]
+
+  @from:cucumber
+  Scenario: process IC with completeInvoices=N should result in Invoice docstatus=IP and IC should be invoiceable again after voiding Invoice
```

Comment:
please obtain test-ids from daniela an tag the scenarios with them (e.g. see `src/test/resources/de/metas/cucumber/features/commissioncontracts/tradeMarginCommissionContract.feature`)